.changeset/stale-mangos-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/code-block': patch
+'@twilio-paste/core': patch
```

Comment:
You can merge a lot of these changesets into a single one, with the same message:

ie:

```
---
'@twilio-paste/code-block': patch
'@twilio-paste/chat-log': patch
'@twilio-paste/core': patch
---

Improved TsDoc type annotations
```

The benefit is, I suspect, it wont bump core 10+ times

Reply:
It'll only bump it once, it rolls core up into a single minor or patch. The advantage here is that you get a package level dedicated changelog entry. So you're not sharing a single changelog entry for multiple packages, making the package changelog much more isolated and specific to that package only.

Reply:
Good to know